tethys_apps/utilities.py

```diff
@@ -574,3 +608,26 @@ def get_all_submodules(m):
     except AttributeError:
         pass
     return modules
+
+
+def delete_secrets(app_name):
+    TETHYS_HOME = Path(get_tethys_home_dir())
+    secrets_yaml_file = TETHYS_HOME / "secrets.yml"
+    portal_secrets = {}
+    if secrets_yaml_file.exists():
+        with secrets_yaml_file.open("r") as secrets_yaml:
+            portal_secrets = yaml.safe_load(secrets_yaml) or {}
+            if "secrets" in portal_secrets:
+                if app_name not in portal_secrets["secrets"]:
+                    # always reset on a new install
+                    portal_secrets["secrets"][app_name] = {}
+                if (
+                    "custom_settings_salt_strings"
+                    in portal_secrets["secrets"][app_name]
+                ):
+                    portal_secrets["secrets"][app_name][
+                        "custom_settings_salt_strings"
+                    ] = {}
+
+                with secrets_yaml_file.open("w") as secrets_yaml:
+                    yaml.dump(portal_secrets, secrets_yaml)
```

Comment:
Another case where a function for getting values out of the secrets.yml file would be good to use to make this more DRY.